### DISCUSSION TITLE
Problems with translation plugin

### DISCUSSION THREAD

**Discussion by BeckeBauer** [kimai/kimai#4047] [Plugins]
I want to change terms in another language than English. However, without knowing the English equivalent, it seems not possible:

![image](https://github.com/kimai/kimai/assets/51003649/14948021-367f-4ed3-87ef-af4f0f42a2d3)

Is there anything I am missing? How can I find the (non-English) term I want to substitute with my own translation?

**Comment by BeckeBauer** [comment]
Managed to find it: One need to firstly change the language of the user to the target language (in my case to German - but **not** German - Germany). Then, the term in the selected language (plus the English term) is shown.

**Comment by kevinpapst** [comment]
I know the current solution is not ideal. It's just hard to build a logical UI for that language switch.
If you have an idea, please share!

BTW: https://github.com/kevinpapst/kimai-plugins

**Comment by BeckeBauer** [comment]
No problem, just a hint in the docs as to how to use the plugin would be helpful (I switched to German (Germany) which did not work and only after I tested it in the demo, I found out that German - without any specific country - has to be selected as language for the user (not in the translation drop-down menu) in order to have the plugin show the pre-translated terms in the user language). 

I do not know the logic behind the selected user language and having the pre-translated terms shown. But when creating an own translation, it would feel more naturally if the selection of the language in the first drop down menu of the plugin would trigger the display of the pre-translated terms in this language (rather than having to switch the language for the user). As example, in my case, selecting "Deutsch" or "Deutsch / Deutschland" in the first drop down menu should trigger the display of terms translated into German even if selected user language is / remains English.

**Reply by kevinpapst** [reply]
Thanks 👍 I updated the docs.
